### PR TITLE
[cisco_meraki] Improved timezone offset error handling

### PR DESCRIPTION
**Pull request by ShourieG**
## Type of change
- Enhancement

## What does this PR do?
Helps improve error handling of timezone field w.r.t the presence of the original time field **_temp.ts_nano** in the ingest pipeline.

## Checklist
- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues
- Relates [IInvalid ID for ZoneOffset when ingesting data via Elastic Serverless Forwarder](https://github.com/elastic/sdh-beats/issues/2824)

### CODE REVIEW

File: packages/cisco_meraki/data_stream/log/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ processors:
     formats:
       - UNIX
     timezone: '{{{_conf.tz_offset}}}'
-    if: 'ctx._conf?.tz_offset != null && ctx._conf.tz_offset != "local"'
+    if: 'ctx._conf?.tz_offset == null || ctx._conf.tz_offset == "local"'
```
Comment (kcreddy) [inline packages/cisco_meraki/data_stream/log/elasticsearch/ingest_pipeline/default.yml:19]: With this condition, this `date` processor will run only if `_conf.tz_offset` is not provided or explicitly set to `null` or `local`. For any other timezone value, even the default `UTC`, the `date` processor itself will not run. 
Have you tested this using system tests, without providing `_conf.tz_offset` field?

File: packages/cisco_meraki/data_stream/log/elasticsearch/ingest_pipeline/default.yml [hunk 2]
```diff
@@ -16,7 +16,6 @@ processors:
     formats:
       - UNIX
     timezone: '{{{_conf.tz_offset}}}'
-    if: 'ctx._conf?.tz_offset != null && ctx._conf.tz_offset != "local"'
```
Comment (srilumpa) [inline packages/cisco_meraki/data_stream/log/elasticsearch/ingest_pipeline/default.yml:19]: If i understand well, removing this condition will simply revert #4811.

Would simply adding after this processor a `date` processor with no `timezone` settings and a `if` condition sets to `ctx._conf?.tz_offset == null || ctx._conf.tz_offset == "local"` be enough?

File: packages/cisco_meraki/data_stream/log/elasticsearch/ingest_pipeline/default.yml [hunk 3]
```diff
@@ -21,6 +21,15 @@ processors:
       - append:
           field: error.message
           value: 'failed to parse time field ({{{ _temp.ts_nano }}}): {{{ _ingest.on_failure_message }}}'
+- date:
+    field: _temp.ts_nano
+    formats:
+      - UNIX
+    if: 'ctx._conf?.tz_offset == null || ctx._conf.tz_offset == "local"'
```
Comment (kcreddy) [inline packages/cisco_meraki/data_stream/log/elasticsearch/ingest_pipeline/default.yml:28]:
```suggestion
    if: 'ctx._conf?.tz_offset == null || ctx._conf?.tz_offset == "local"' 
```

**Review by kcreddy — APPROVED** [review]
Minor change. Rest LGTM

**Pull request merged**